langserve/server.py

```diff
@@ -333,6 +335,52 @@ def _get_base_run_id_as_str(
         raise AssertionError("No run_id found for the given run")
 
 
+def _json_encode_response(obj: BaseModel, *, is_batch: bool = False) -> JSONResponse:
+    """Return a JSONResponse with the given content."""
```

Comment:
Can we comment that we're doing this (I assume) because of pydantic compatibility issues, so we have to respond in JSON rather than a set type?